lib/wiki_discouraged_article_manager.rb

```diff
@@ -9,12 +9,18 @@ def retrieve_wiki_edu_discouraged_articles
   private
 
   def update_or_create_discouraged_articles
-    Category.find_or_create_by(
-      name: ENV['blocked_assignment_category'],
-      wiki_id: en_wiki.id,
-      source: 'category',
-      depth: 0
-    ).refresh_titles
+    blocked_assignment_category = ENV['blocked_assignment_category']
+
```

Comment:
I'd prefer a `return unless` guard clause here, rather than the branching `if` statement.